scico/linop/radon_svmbir.py

```diff
@@ -24,12 +32,6 @@
         "for instructions on how to install the SVMBIR."
```

Comment:
For grammatical and consistency reasons, change to "... install svmbir", or even better, simply "... how to install it."

Reply:
Done